README.md

```diff
@@ -94,23 +94,26 @@ val featureAbc = ExampleRemoteBoolean("feature_abc_disabled", true) // WRONG! tr
 
 ### View IDs
 
-When Kotlin extensions are used (and view IDs are made available via `import`), view ID names
-(`android:id` in XML) should be all lowercase with underscores separating words (snake_case). It is
-required that every name have at least one underscore to distinguish it from local variables. In
-instances were it proves difficult, then name may be postfixed with the component type
-(e.g. `_button` or `_view`).
+View IDs (`android:id` in XML) should be all lowercase with underscores separating words ([snake_case]), for example:
 
 ```xml
 <android.support.v7.widget.RecyclerView
     android:id="@+id/dance_moves"
     />
 ```
 
+View IDs should **not** be suffixed with the view type (e.g. `_button` or `_view`) unless needed to distinguish between
+two identically named views which differ only by their view type, for example:
 
-```kotlin
-import kotlinx.android.synthetic.main.scan.dance_moves
-import kotlinx.android.synthetic.main.scan.login_button
-import kotlinx.android.synthetic.main.scan.root_view
+```xml
+<TextView
```

Comment:
Dreading having to do a pass on the app project to make these same changes though it's probably past due

Reply:
Ya, it will take some time, but we can slowly chip away at it over time.